include/lcp-templater.php

```diff
@@ -107,6 +107,10 @@ public static function get_templates() {
     $paths = self::get_template_paths();
 
     foreach ($paths as $path) {
+      if (!is_dir($path)) {
```

Comment:
This is not necessary, the call to `get_template_paths()` above already checks this